components/Nav.js

```diff
@@ -1,4 +1,6 @@
 import Hamburger from "./Hamburger";
+import Social from "./Social-icons.js";
```

Comment:
```suggestion
import Social from "./Social.js";
```

You'd have to change the filename too, but I think `import <Name> from "./<Name>.js";` is going to help you in the long run and your colleagues will appreciate it.